prow/plugins/updateconfig/updateconfig.go

```diff
@@ -141,11 +142,11 @@ func handle(gc githubClient, kc kubeClient, log *logrus.Entry, pre github.PullRe
 	case 0:
 		return nil
 	case 1:
-		msg = fmt.Sprintf("Updated the %s configmap", updated[0])
+		msg = fmt.Sprintf("Updated the %s configmap on namespace %s", updated[0].Name, updated[0].Namespace)
 	default:
 		msg = fmt.Sprintf("Updated the following %d configmaps:\n", n)
 		for _, cm := range updated {
-			msg += fmt.Sprintf("  * %s\n", cm)
+			msg += fmt.Sprintf("  * %s on namespace %s\n", cm.Name, cm.Namespace)
```

Comment:
Here and above:
s/on/in/

This won't display nicely in the common case where `cm.Namespace == ""`.